docs/server/settings.rst

```diff
@@ -294,20 +298,50 @@ Translation environment configuration settings.
             'HOST': 'localhost',
```

Comment:
Lets try to see this from a different angle. If we merge both management commands that means that you you will be able to specify the TM using `--tm` and since the settings do not enforce in any way that it can only be a single local TM, then having `local` instead of `default` is actually not contributing anything either. At least `default` is now treated as the default TM.